core/libs/bridges/setup.py

```diff
@@ -12,7 +12,7 @@
 
 setuptools.setup(
     name="bridges",
-    version="0.1.0",
```

Comment:
you need now to update all services that depend of bridges to use the new version, or just remove this commit.

Reply:
The version number probably should change, but since the API isn't changing at all I don't think it's worth the effort of finding where bridges is being used and updating those version numbers in the requirements. I've removed the commit.